config/default/manager_auth_proxy_patch.yaml

```diff
@@ -26,9 +26,6 @@ spec:
           protocol: TCP
           name: https
         resources:
-          limits:
-            cpu: 500m
-            memory: 128Mi
```

Comment:
it is good practice to keep the memory limits, you can increase it, but lets not completely drop it

Reply:
You can remove the cpu limit, it is a good practice actually, just the memory that we should set because it impacts the stability of the node of the container has a bug and tries to use all the memory.

Reply:
updated